ssg/controls.py

```diff
@@ -21,14 +64,18 @@ def __init__(self):
         self.title = ""
         self.description = ""
         self.automated = ""
+        self.status = None
 
     @classmethod
     def from_control_dict(cls, control_dict, env_yaml=None, default_level=["default"]):
         control = cls()
         control.id = ssg.utils.required_key(control_dict, "id")
         control.title = control_dict.get("title")
         control.description = control_dict.get("description")
-        control.automated = control_dict.get("automated", "yes")
+        control.status = Status.from_control_info(control.id, control_dict.get("status", None))
+        control.automated = control_dict.get("automated", "no")
+        if control.status == "automated":
+            control.automated = "yes"
```

Comment:
I wonder if we should proceed with  removal of  the `automated` key from the controls.
@matejak Thoughts?

Reply:
I have asked the community on Gitter regarding naming. I don't plan to spend too much time on bikeshedding, but let's try to get naming right.